proxy/http2/Http2Stream.cc

```diff
@@ -671,7 +645,26 @@ Http2Stream::signal_read_event(int event)
     if (this->_read_vio_event) {
       this->_read_vio_event->cancel();
     }
-    this->_read_vio_event = this_ethread()->schedule_imm(read_vio.cont, event, &read_vio);
+    this->_read_vio_event = this_ethread()->schedule_in(this, retry_delay, event, &read_vio);
+  }
+}
+
+void
+Http2Stream::signal_write_event(int event)
+{
+  if (this->write_vio.cont == nullptr || this->write_vio.cont->mutex == nullptr || this->write_vio.op == VIO::NONE) {
+    return;
+  }
+
+  MUTEX_TRY_LOCK(lock, write_vio.cont->mutex, this_ethread());
+  if (lock.is_locked()) {
+    inactive_timeout_at = Thread::get_hrtime() + inactive_timeout;
+    this->write_vio.cont->handleEvent(event, &this->write_vio);
+  } else {
+    if (this->_write_vio_event) {
+      this->_write_vio_event->cancel();
+    }
+    this->_write_vio_event = this_ethread()->schedule_in(this, retry_delay, event, &write_vio);
```

Comment:
If I understand this correctly, we don't need to pass write_vio because it always use THE write_vio that Http2Stream has, right?

Is retry_delay required? Should we change `signal_read_event` to use it too (on maybe another PR)?